dmcontent/content_loader.py

```diff
@@ -199,7 +205,7 @@ def get_question_as_section(self, question_slug):
             editable=self.edit_questions,
             edit_questions=False,
             questions=question.get('questions', [question]),
-            description=question.get('hint'),
+            description=question.get('hint') if question.get('questions') else '',
```

Comment:
Yes, otherwise when you get a question as a section the hint gets duplicated and emphasised in a bad way.